xml/Sector.xml

```diff
@@ -3526,8 +3669,8 @@
                 <narrative xml:lang="fr">Assistance alimentaire d'urgence</narrative>
             </name>
             <description>
-                <narrative>Provision and distribution of food; cash and vouchers for the purchase of food; non-medical nutritional interventions for the benefit of crisis-affected people, including refugees and internally displaced people in developing countries in emergency situations. Includes logistical costs. Excludes non-emergency food assistance (52010), food security policy and administrative management (43071), household food programmes (43072) and medical nutrition interventions (therapeutic feeding) (72010).</narrative>
-                <narrative xml:lang="fr">Fourniture et distribution de nourriture; de l'argent et des bons pour l'achat de nourriture; interventions nutritionnelles non médicales au profit des personnes touchées par la crise, notamment des réfugiés et des personnes déplacées à l'intérieur de leur propre pays, dans des pays en développement en situation d'urgence. Comprend les coûts logistiques. Sont exclus l'assistance alimentaire non urgente (52010), la politique de sécurité alimentaire et la gestion administrative (43071), les programmes alimentaires des ménages (43072) et les interventions de nutrition médicale (alimentation thérapeutique) (72010).</narrative>
+                <narrative>Provision and distribution of food; cash and vouchers for the purchase of food; non-medical nutritional interventions for the benefit of crisis-affected people, including refugees and internally displaced people in developing countries in emergency situations. Includes logistical costs. Excludes non-emergency food assistance (52010), food security policy and administrative management (43071), household food programmes (43072) and medical nutrition interventions (therapeutic feeding) (72010 and 72011).</narrative>
+                <narrative xml:lang="fr">Fourniture et distribution de nourriture; de l'argent et des bons pour l'achat de nourriture; interventions nutritionnelles non médicales au profit des personnes touchées par la crise, notamment des réfugiés et des personnes déplacées à l'intérieur de leur propre pays, dans des pays en développement en situation d'urgence. Comprend les coûts logistiques. Sont exclus l'assistance alimentaire non urgente (52010), la politique de sécurité alimentaire et la gestion administrative (43071), les programmes alimentaires des ménages (43072) et les interventions de nutrition médicale (alimentation thérapeutique) (72010 et 72011).</narrative>
```

Comment:
Description change (adding reference to new code 72011) as part of Feb 2020 update